### PR TITLE
PEP8 / Typos

### PR DESCRIPTION
**Pull request by slowr**
- [x] PEP8 Compliant
- [x] Removed unused imports
- [x] Fixed typos

### CODE REVIEW

File: lib/traixroute/controller/string_handler.py [hunk 1]
```diff
@@ -284,7 +305,6 @@ def shortinlong(self, sname, lname):
 
         if sname == '' or lname == '':
             return False
-        whitelist = set('abcdefghijklmnopqrstuvwxyz1234567890 ')
```
Comment (slowr) [inline lib/traixroute/controller/string_handler.py:287]: this line was not used

File: lib/traixroute/detector/detection_rules.py [hunk 2]
```diff
@@ -361,26 +401,28 @@ def check_edges(self, rule, path_asn, current, str_to_chk, ixp_long, ixp_short):
...
 
         if final1 == '' or final2 == '':
             return True
-        if self.is_int(final1) and self.is_int(final2) and 'AS_M' in str_chk:
-            if (final1 == final2 and path_asn[current - 1] != path_asn[current + 1]) or (final1 != final2 and path_asn[current - 1] == path_asn[current + 1]):
+        if self.is_int(final1) and self.is_int(final2) and 'AS_M' in str_to_chk:
+            if (final1 == final2 and path_asn[current - 1] != path_asn[current + 1]) or (
+                    final1 != final2 and path_asn[current - 1] == path_asn[current + 1]):
                 return False
-        elif self.is_int(final1) and self.is_int(final2) and 'IXP_IP' in str_chk:
```
Comment (slowr) [inline lib/traixroute/detector/detection_rules.py:375]: `str_chk` was undefined

File: lib/traixroute/detector/detection_rules.py [hunk 3]
```diff
@@ -361,26 +401,28 @@ def check_edges(self, rule, path_asn, current, str_to_chk, ixp_long, ixp_short):
...
-            string_hanlde = string_handler.string_handler()
```
Comment (slowr) [inline lib/traixroute/detector/detection_rules.py:376]: `string_hanlde` typo which was used as `string_handle` below

File: lib/traixroute/handler/handle_pdb.py [hunk 4]
```diff
@@ -187,40 +186,44 @@ def extract_pfx(self, json_pfx, ixlan_dict, id_to_names, reserved_tree, region_d
...
+                            subnet2region[subnet] = region_dict[ix_id]
+
+                        elif subnet in pfxs_dict:
                             assign_tuple = []
-                            for IXP in pfx_dict[subnet]:
+                            for IXP in pfxs_dict[subnet]:
                                 assign_tuple = assign_tuple + \
-                                    handler.assign_names(IXP[1], id_to_names[ix_id][1], IXP[0], id_to_names[ix_id][0])
-                            pfxs_dict[subnet]       = assign_tuple
-                            subnet2regions[subnet]  = region_dict[ix_id]
```
Comment (slowr) [inline lib/traixroute/handler/handle_pdb.py:216]: typo `subnet2regions`

File: lib/traixroute/application.py [hunk 5]
```diff
@@ -54,135 +49,155 @@ class traIXroute():
...
+
     def main(self):
         '''
         The main function which calls all the other traIXroute modules.
         '''
-        
+
         # Calls the parser to analyze the command line arguments.
         self.traixparser.parse_input()
-        inputIP             = self.traixparser.inputIP
```
Comment (gnomikos) [inline lib/traixroute/application.py:155]: I would prefer to keep the initial aligned structure

File: lib/traixroute/application.py [hunk 6]
```diff
@@ -54,135 +49,155 @@ class traIXroute():
...
-        self.db_extract     = None
-        self.enable_stats   = None
-        self.exact_time     = None
-        self.ripe           = None
-        self.selected_tool  = None
-        self.ripe_handle    = None
-        self.traixparser    = traixroute_parser.traixroute_parser(self.version)
-        self.detection_rules= detection_rules.detection_rules()
-        self.json_handle    = handle_json.handle_json()
+        self.version = '2.3'
```
Comment (gnomikos) [inline lib/traixroute/application.py:52]: I would prefer to keep the initial aligned structure

File: lib/traixroute/controller/traixroute_output.py [hunk 7]
```diff
@@ -38,34 +38,34 @@ class traixroute_output():
...
     def __init__(self):
-        
+
         self.measurement_json = {
             'ixp_crossings': [],
             'remote_peering': [],
             'possible_ixp_crossings': []
         }
 
-        self.measurement_info   = ''
```
Comment (gnomikos) [inline lib/traixroute/controller/traixroute_output.py:48]: I would prefer to keep the initial aligned structure

File: lib/traixroute/controller/traixroute_output.py [hunk 8]
```diff
@@ -76,54 +76,66 @@ def flush(self, traixparser):
             del self.measurement_json['possible_ixp_crossings']
 
         # SupplementS the relative output lists.
-        if not traixparser.flags['silent']      : print(output)
```
Comment (gnomikos) [inline lib/traixroute/controller/traixroute_output.py:79]: I would prefer to keep the initial aligned structure

File: lib/traixroute/controller/traixroute_output.py [hunk 9]
```diff
@@ -76,54 +76,66 @@ def flush(self, traixparser):
...
+            g) dirty_ips: The number of dirty ips.
             h) additional_ip2asn: A dictionary with {IXP IP}=[ASN] specified by the user.
             i) additional_subnet2name: A dictionary with {IXP Subnet}=[IXP long name, IXP short name] specified by the user.
             j) lenreserved: The number of the imported reserved subnets.
-            k) db_print: Flag to output the db stats. 
+            k) db_print: Flag to output the db stats.
             l) mypath: The path to the database.
         '''
 
-        tmp =  'Imported '  + str(lenreserved) + ' Reserved Subnets.\n'
```
Comment (gnomikos) [inline lib/traixroute/controller/traixroute_output.py:111]: I would prefer to keep the initial aligned structure

File: lib/traixroute/detector/detection_rules.py (hunk 2 above)
Comment (gnomikos) [reply lib/traixroute/detector/detection_rules.py:375]: ok

File: lib/traixroute/detector/detection_rules.py (hunk 3 above)
Comment (gnomikos) [reply lib/traixroute/detector/detection_rules.py:376]: I wonder why with this typo code still works

File: lib/traixroute/controller/string_handler.py (hunk 1 above)
Comment (gnomikos) [reply lib/traixroute/controller/string_handler.py:287]: keep it for now, I have to check if we still need string sanitization. You can add a related issue.

File: lib/traixroute/handler/database_extract.py [hunk 10]
```diff
@@ -166,56 +165,86 @@ def dbextract(self):
             else:
                 print("Loading from database.")
 
-            user_imports    = handle_complementary.extract_additional_info()
```
Comment (gnomikos) [inline lib/traixroute/handler/database_extract.py:169]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/database_extract.py [hunk 11]
```diff
@@ -147,12 +146,12 @@ def dbextract(self):
                 flag = flag or item.result()[1]
 
             if not flag:
-                self.final_ixp2asn      = results[0].result()[0]
```
Comment (gnomikos) [inline lib/traixroute/handler/database_extract.py:150]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/database_extract.py [hunk 12]
```diff
@@ -166,56 +165,86 @@ def dbextract(self):
...
+            user_imports = handle_complementary.extract_additional_info()
+            pch = handle_pch.pch_handle(self.downloader, self.libpath)
+            peeringdb = handle_pdb.peering_handle(
+                self.downloader, self.libpath)
+            dict_merge = dict_merger.dict_merger()
+            asn_hand_info = handle_complementary.asn_memb_info()
 
             user_imports.extract_additional_info(self.homepath)
             # additional IXP membership information.
-            additional_ixp_ip2asn       = user_imports.ixp_ip2asn
```
Comment (gnomikos) [inline lib/traixroute/handler/database_extract.py:177]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/database_extract.py [hunk 13]
```diff
@@ -166,56 +165,86 @@ def dbextract(self):
...
-            additional_info_help_tree   = user_imports.additional_info_help_tree
-            additional_pfx2cc           = user_imports.pfx2cc
+            additional_subnet2name = user_imports.Subnet
+            additional_info_tree = user_imports.additional_info_tree
+            additional_info_help_tree = user_imports.additional_info_help_tree
+            additional_pfx2cc = user_imports.pfx2cc
 
             results = []
             with concurrent.futures.ThreadPoolExecutor(max_workers=3) as executor:
-                results.append(executor.submit(asn_hand.routeviews_extract, self.reserved_sub_tree))
```
Comment (gnomikos) [inline lib/traixroute/handler/database_extract.py:186]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/database_extract.py [hunk 14]
```diff
@@ -166,56 +165,86 @@ def dbextract(self):
...
-                results.append(executor.submit(peeringdb.peering_handle_main, self.reserved_sub_tree, country2cc))
-                results.append(executor.submit(pch.pch_handle_main, self.reserved_sub_tree, additional_info_tree, country2cc))
-
-            self.asn_routeviews     = results[0].result()[0]
```
Comment (gnomikos) [inline lib/traixroute/handler/database_extract.py:190]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/dict_merger.py [hunk 15]
```diff
@@ -108,22 +109,22 @@ def merge_ixp2asns(self, d1, d2, flag, Subnet_tree, replace=False):
...
+
         if replace:
             for key in d1:
                 d2[key] = d1[key]
-            return d2 
-            
+            return d2
+
         if len(d2) > len(d1):
-            big     = d2
```
Comment (gnomikos) [inline lib/traixroute/handler/dict_merger.py:121]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/dict_merger.py [hunk 16]
```diff
@@ -108,22 +109,22 @@ def merge_ixp2asns(self, d1, d2, flag, Subnet_tree, replace=False):
...
-            small   = d1
+            big = d2
+            small = d1
         else:
-            big     = d1
```
Comment (gnomikos) [inline lib/traixroute/handler/dict_merger.py:124]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/dict_merger.py [hunk 17]
```diff
@@ -149,28 +153,27 @@ def merge_ixp2asns(self, d1, d2, flag, Subnet_tree, replace=False):
...
             a) d1,d2: The two dictionaries to be merged.
         Output:
             a) d1 or d2: The output dictionary after merging.
         '''
-        
-        handle = string_handler.string_handler()
-        
+
         if len(d2) > len(d1):
-            big     = d2
```
Comment (gnomikos) [inline lib/traixroute/handler/dict_merger.py:164]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/dict_merger.py [hunk 18]
```diff
@@ -149,28 +153,27 @@ def merge_ixp2asns(self, d1, d2, flag, Subnet_tree, replace=False):
...
-            small   = d1
+            big = d2
+            small = d1
         else:
-            big     = d1
```
Comment (gnomikos) [inline lib/traixroute/handler/dict_merger.py:167]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/handle_complementary.py [hunk 19]
```diff
@@ -35,9 +34,9 @@ class asn_handle():
 
     def __init__(self, downloader, libpath):
         self.route_filename = '/RouteViews/routeviews'
-        self.downloader     = downloader
```
Comment (gnomikos) [inline lib/traixroute/handler/handle_complementary.py:38]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/handle_complementary.py [hunk 20]
```diff
@@ -305,49 +318,58 @@ def extract_additional_info(self, mypath):
...
                                 int(line_split[1])
-                            except:
-                                print('additional_info.txt: Invalid syntax in line ' + str(i) + '. Exiting.')
+                            except BaseException:
+                                print(
+                                    'additional_info.txt: Invalid syntax in line ' +
+                                    str(i) +
+                                    '. Exiting.')
                                 sys.exit(0)
-                            asn             = line_split[1]
```
Comment (gnomikos) [inline lib/traixroute/handler/handle_complementary.py:341]: I would prefer to keep the initial alignment

File: lib/traixroute/handler/handle_pdb.py (hunk 4 above)
Comment (gnomikos) [reply lib/traixroute/handler/handle_pdb.py:216]: ok

**Pull request closed without merging**